pkg/builder/builder.go

```diff
@@ -368,6 +370,65 @@ func (b generalBuilder) checkIfNeedBuild(ctx context.Context) bool {
 	return true
 }
 
+func getDepsFiles(deps []string) []string {
+	tHandle := reflect.TypeOf(*ir.DefaultGraph)
+	vHandle := reflect.ValueOf(*ir.DefaultGraph)
+	deps = searchFileInGraph(tHandle, vHandle, deps)
+	return deps
+}
+
+// Match all filed in ir.Graph with the given keyword
+func likeFileFiled(str string) bool {
+	nameKeyword := []string{
+		"File",
```

Comment:
Will this also match `HTTPInfo.Filename`? That one is an alias for the downloaded file.

Reply:
Would it better using struct tag?

Reply:
> Will this also match `HTTPInfo.Filename`? That one is an alias for the downloaded file.

I think it matched, let me try it.

Reply:
I think this will match `HTTPInfo.Filename`. That's not a file on the host. So the function `checkDepsFileUpdate` will always return **True**. Is that an expected behavior?